frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedColumns.jsx

```diff
@@ -57,12 +56,10 @@ export default class ChartSettingOrderedColumns extends Component {
     onChange(columnSettings);
   };
 
-  getColumnName = columnSetting =>
-    getFriendlyName(
-      findColumnForColumnSetting(this.props.columns, columnSetting) || {
-        display_name: "[Unknown]",
-      },
-    );
+  getColumnName = columnSetting => {
+    const { getColumnName } = this.props;
+    return getColumnName(columnSetting) || "[Unknown]";
```

Comment:
This should never actually happen since if we have a column setting that can't be found in `data.cols`, the settingDef's `isValid()` check should return false and we should recompute a default, which is based off data.cols. However, I figured I should keep a backup value in there anyways.